### PR TITLE
Fix abbreviations of Litre/min and milliLitre/min

### PR DESCRIPTION
**Pull request by flopp999**
liter per minute and millilitre per minute

### CODE REVIEW

File: public/app/core/utils/kbn.ts [hunk 1]
```diff
@@ -584,8 +584,8 @@ kbn.valueFormats.flowcms = kbn.formatBuilders.fixedUnit('cms');
 kbn.valueFormats.flowcfs = kbn.formatBuilders.fixedUnit('cfs');
 kbn.valueFormats.flowcfm = kbn.formatBuilders.fixedUnit('cfm');
 kbn.valueFormats.litreh = kbn.formatBuilders.fixedUnit('l/h');
-kbn.valueFormats.flowlpm = kbn.formatBuilders.decimalSIPrefix('L');
-kbn.valueFormats.flowmlpm = kbn.formatBuilders.decimalSIPrefix('L', -1);
+kbn.valueFormats.flowlpm = kbn.formatBuilders.decimalSIPrefix('L/min');
```
Comment (marefr) [inline public/app/core/utils/kbn.ts]: Shouldn't L be a small letter?
```suggestion
kbn.valueFormats.flowlpm = kbn.formatBuilders.decimalSIPrefix('l/min');
```

File: public/app/core/utils/kbn.ts [hunk 2]
```diff
@@ -584,8 +584,8 @@ kbn.valueFormats.flowcms = kbn.formatBuilders.fixedUnit('cms');
...
+kbn.valueFormats.flowmlpm = kbn.formatBuilders.decimalSIPrefix('L/min', -1);
```
Comment (marefr) [inline public/app/core/utils/kbn.ts]: Shouldn't this be `mL/min`?
```suggestion
kbn.valueFormats.flowmlpm = kbn.formatBuilders.decimalSIPrefix('mL/min', -1);
```

**Review by marefr — CHANGES_REQUESTED** [review]
Some minor comments

File: public/app/core/utils/kbn.ts (hunk 1 above)
Comment (flopp999) [reply public/app/core/utils/kbn.ts]: I do want the small but it was big L for volume so I was using big

File: public/app/core/utils/kbn.ts (hunk 2 above)
Comment (flopp999) [reply public/app/core/utils/kbn.ts]: You are correct, my mistake. 
I donät know if it should be ml/min or mL/min?

Comment (marefr) [reply public/app/core/utils/kbn.ts]: I'm not sure either. Looking at the submenu title it's `mL/min`.

**Review by marefr — APPROVED** [review]

**Comment by marefr** [comment]
Thank you for contributing to Grafana!

**Pull request merged**